Sygic.Corona.Application/Commands/SendMfaTokenCommandHandler.cs

```diff
@@ -38,7 +38,9 @@ public async Task<SendMfaTokenResponse> Handle(SendMfaTokenCommand request, Canc
 
             try
             {
-                string message = $"Vas Secure kod: {profile.AuthToken}. Covid-19 App";
+                string message = !string.IsNullOrEmpty(request.Message) 
+                    ? string.Format(request.Message, profile.AuthToken)
```

Comment:
You should handle the case of wrong message format (no {days} in the message)

Reply:
string.Format won't throw an exception if the message has no {days} argument.

Reply:
string.Format won't throw an exception if the message has no {days} argument, and it SHOULD throw an exception if the string is not correctly formatted.